src/index.tsx

```diff
@@ -4,9 +4,22 @@ import "./index.css";
 import App from "./App";
 import * as serviceWorker from "./serviceWorker";
 
-ReactDOM.render(<App />, document.getElementById("root"));
+// @ts-ignore
+const render = (Component: typeof App) =>
+  ReactDOM.render(<Component />, document.getElementById("root"));
 
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
 serviceWorker.unregister();
+
+render(App);
+
+// @ts-ignore
+if (module.hot) {
```

Comment:
What's this section for? Guessing its trying to fix a hot-reload issue?

Whatever the reason, there should be a comment here explaining it, since its ignoring compiler errors and changing a pretty common React pattern.

Reply:
`hot` isn't defined on `module` types, so Typescript complains.

Reply:
Some Googling indicates we can install `@types/webpack-env` for that.  Dunno if you want to bother.

Reply:
For a single section, I don't think it's worth installing a package for. I've added a comment explaining the situation instead: fcaa7cf